src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.cs

```diff
@@ -31,7 +28,8 @@ internal partial class CSharpSimplificationService : AbstractSimplificationServi
                 new CSharpParenthesesReducer(),
                 new CSharpEscapingReducer(),
                 new CSharpMiscellaneousReducer(),
-                new CSharpInferredMemberNameReducer());
+                new CSharpInferredMemberNameReducer(),
+                new CSharpDefaultExpressionReducer());
```

Comment:
It does make me wonder what happens when a user prefers `var` and also prefers `default`, and you ask it to simplify this:

```csharp
int x = default(int);
```